sklearn/tests/test_metaestimators.py

```diff
@@ -44,6 +49,38 @@ def __init__(self, name, construct, skip_methods=(),
                                 'predict'])
 ]
 
+TESTED_META = [
```

Comment:
Why if this better here than in test_pipeline.py, test_from_model.py, etc?

Reply:
I'm not sure it is. Having them in one place has pros and cons I think.
Right now it shows what kind of cases we're testing and what interactions we're testing.

Reply:
I have no strong opinion either way, I was about to move them to the respective files, mostly into the pipeline tests.